doc/source/feature_comparison.rst

```diff
@@ -20,111 +20,260 @@
-+========================================================+======================+==============+
-|Configurable Page Sizes                                 | Yes                  | Yes          |
-+--------------------------------------------------------+----------------------+--------------+
-|Configurable Fast Index Creation                        | Yes                  |              |
-+--------------------------------------------------------+----------------------+--------------+
-|Changed Page Tracking                                   | Yes                  |              |
-+--------------------------------------------------------+----------------------+--------------+
-|PAM Authentication                                      | Yes                  | Yes [#n-1]_  |
-+--------------------------------------------------------+----------------------+--------------+
-|Threadpool                                              | Yes                  | Yes [#n-1]_  |
-+--------------------------------------------------------+----------------------+--------------+
-|Backup Locks                                            | Yes                  |              |
-+--------------------------------------------------------+----------------------+--------------+
-|Extended ``SHOW GRANTS``                                | Yes                  |              |
-+--------------------------------------------------------+----------------------+--------------+
-|Improved Handling of Corrupted Tables                   | Yes                  |              |
-+--------------------------------------------------------+----------------------+--------------+
-|Ability to Kill Idle Transactions                       | Yes                  |              |
-+--------------------------------------------------------+----------------------+--------------+
-|START TRANSACTION WITH CONSISTENT SNAPSHOT improvements | Yes                  |              |
-+--------------------------------------------------------+----------------------+--------------+
-
-
-+---------------------------------------+----------------------+--------------+
-|Running Database as a Service          |Percona Server 5.7.10 |MySQL 5.7.10  |
-+=======================================+======================+==============+
-|Special Utility User                   | Yes                  |              |
-+---------------------------------------+----------------------+--------------+
-|Expanded Program Option Modifiers      | Yes                  |              |
-+---------------------------------------+----------------------+--------------+
-|Enforcing the Specific Storage Engine  | Yes                  |              |
-+---------------------------------------+----------------------+--------------+
+.. tabularcolumns:: |p{5cm}|p{5cm}|p{5cm}|
+
+.. list-table::
+    :header-rows: 1
+
+    * - Features
+      - Percona Server 5.7.24
```

Comment:
Shouldn't this be .27 now?

Reply:
The information for the PS Feature Comparison was for PS 5.7.24 compared to MySQL 5.7.24.

Reply:
And did anything change since then? I don't think so, upstream only did bugfixes, same for us. So I would go with .27, since that's the latest version. @georgelorchpercona ?

Reply:
Agreed, I don't think anything changed so should probably be 5.7.27

Reply:
updated to 5.7.27